### ISSUE TITLE
Secondary navigation dropdown menu should expand on :hover or :focus

### ISSUE THREAD

**Issue by djcelaya** [az-digital/arizona-bootstrap#1827]
The current implementation requires the user to click on the dropdown-toggle to expand the submenu. MarCom would like it to expand on hover instead. To maintain accessibility compliance, the submenu should also expand on `:focus`, which is gained via keyboard navigation.
<img width="483" height="286" alt="Image" src="https://github.com/user-attachments/assets/a635a4dd-daa9-43dd-afd1-ab4fb1af4a81" />

## Conditions of satisfaction
- [x] When a user hovers over a dropdown-toggle at the primary level, the corresponding secondary level dropdown menu should be shown.
- [x] When a dropdown-toggle gains `:focus` via keyboard navigation at the primary level, the corresponding secondary level dropdown menu should be shown.
